assets/js/googlesitekit/notifications/datastore/notifications.js

```diff
@@ -393,6 +495,32 @@ export const resolvers = {
 };
 
 export const selectors = {
+	/**
+	 * Gets all seen notifications view date arrays.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {Object} Object with notification IDs as keys and view count as values.
```

Comment:
This is incorrect; the value should be the array of dates this notification was viewed on, according to the data in the store.

```suggestion
	 * @return {Object} Object with notification IDs as keys and array of dates viewed as the value.
```